conans/client/rest/rest_client.py

```diff
@@ -12,7 +12,7 @@ class RestApiClient(object):
         Rest Api Client for handle remote.
     """
 
-    def __init__(self, output, requester, put_headers=None):
+    def __init__(self, output, requester, block_v2, put_headers=None):
```

Comment:
Please, express it as `revisions_enabled`, it will add more consistence across the codebase.